tests/integration/test_enrollment/test_agent_auth_enrollment.py

```diff
@@ -24,64 +24,124 @@
 
 import pytest
 import os
+import platform
+import subprocess
 
-from wazuh_testing.tools import LOG_FILE_PATH
 from wazuh_testing.tools.configuration import load_wazuh_configurations
 from wazuh_testing.tools.services import control_service
-from wazuh_testing.tools.file import load_tests, truncate_file
-from wazuh_testing.tools.monitoring import FileMonitor, make_callback
-from conftest import *
+from wazuh_testing.tools.file import load_tests
+from wazuh_testing.tools.monitoring import QueueMonitor, make_callback
+from wazuh_testing.tools.utils import get_host_name
+from wazuh_testing.agent import AgentAuthParser
+from wazuh_testing.tools import WAZUH_PATH
+
+AGENT_AUTH_BINARY_PATH = '/var/ossec/bin/agent-auth' if platform.system() == 'Linux' else \
+    os.path.join(WAZUH_PATH, 'agent-auth.exe')
 
 # Marks
 pytestmark = [pytest.mark.linux, pytest.mark.win32, pytest.mark.tier(level=0), pytest.mark.agent]
 
+test_data_path = os.path.join(os.path.dirname(os.path.realpath(__file__)), 'data')
+tests_path = os.path.join(test_data_path, 'wazuh_enrollment_tests.yaml')
+configurations_path = os.path.join(test_data_path, 'wazuh_enrollment_conf.yaml')
+tests = load_tests(tests_path)
 configurations = load_wazuh_configurations(configurations_path, __name__)
```

Comment:
We need to add the ids so the when we execute the test, it is more readable. We could do:
```
configuration_ids = ['agent_auth_enrollment']
test_case_ids = [f"{test_case['name']}" for test_case in tests]
```

And we should add them to the fixtures:
```
@pytest.fixture(scope='module', params=configurations, ids=configuration_ids)
def get_configuration(request):
.
.
.

@pytest.fixture(scope='function', params=tests, ids=test_case_ids )
def get_current_test_case(request):
.
.
.
```

Reply:
Implemented!